build.gradle

```diff
@@ -32,17 +33,18 @@ def dependencyVersions = [
         dropwizard: '1.3.5',
         ida_utils: '337',
         ida_test_utils: '44',
-        hub_saml: "$opensaml-15606",
+        hub_saml: "$opensaml-15701",
         dev_pki: '1.1.0-34',
         trust_anchor: '1.0-34',
-        saml_libs_version: "$opensaml-157"
+        saml_libs_version: "$opensaml-162"
 ]
 
 repositories {
     if (System.getenv('VERIFY_USE_PUBLIC_BINARIES') == 'true') {
         logger.warn('Production builds MUST NOT be built with public binaries.\nUse artifactory/whitelisted-repos for production builds.\n\n')
         maven { url 'https://dl.bintray.com/alphagov/maven' } // For dropwizard-logstash
         maven { url 'https://dl.bintray.com/alphagov/maven-test' } // For other public verify binaries
+        maven { url 'https://build.shibboleth.net/nexus/content/repositories/releases' }
```

Comment:
would be worth adding the comment as with the above repos? that this is for opensaml...

Reply:
Done.